src/utils/update/fetch.beneficial.owners.private.data.ts

```diff
@@ -31,20 +31,18 @@ export const fetchBeneficialOwnersPrivateData = async (appData: ApplicationData,
 };
 
 const mapBeneficialOwnersPrivateData = (boPrivateData, appData: ApplicationData) => {
-  if (boPrivateData === undefined || boPrivateData.length === 0) {
-    return;
-  }
-  const mapUsualResidentialAddress = beneficialOwner => {
+  const mapIndividualOwners = beneficialOwner => {
     if (beneficialOwner.ch_reference) {
-      beneficialOwner.usual_residential_address = mapBoPrivateAddress(boPrivateData, beneficialOwner.ch_reference, false);
+      mapIndividualBOPrivateData(boPrivateData, beneficialOwner);
```

Comment:
I believe this value replaces the mapped value  of 
https://github.com/companieshouse/overseas-entities-web/blob/47a7c715f598289808ab49f8971c142af80434d9/src/utils/update/psc.to.beneficial.owner.type.mapper.ts#L20C5-L20C5